src/solvers/dgsem_tree/dg_1d.jl

```diff
@@ -497,6 +501,39 @@ function calc_boundary_flux_by_direction!(surface_flux_values::AbstractArray{<:A
   return nothing
 end
 
+function calc_boundary_flux_by_direction!(surface_flux_values::AbstractArray{<:Any,3}, t,
+                                          boundary_condition, equations, nonconservative_terms::Val{true},
+                                          surface_integral, dg::DG, cache,
+                                          direction, first_boundary, last_boundary)
+
+    surface_flux, nonconservative_flux = surface_integral.surface_flux
+    @unpack u, neighbor_ids, neighbor_sides, node_coordinates, orientations = cache.boundaries
+
+    @threaded for boundary in first_boundary:last_boundary
+    # Get neighboring element
+    neighbor = neighbor_ids[boundary]
```

Comment:
Please respect the common indentation pattern:
- Indent by two spaces per level
- Indent the body of a `for` loop